css-properties-values-api/Overview.bs

```diff
@@ -596,19 +596,15 @@ Parsing the syntax string {#parsing-syntax}
 			return |descriptor|;
 			otherwise, return failure.
 
-		:   U+007C VERTICAL LINE (|)
+		:   anything else
 		:: If |descriptor|’s [=list/size=] is greater than zero,
-			<a>consume a syntax component</a> from |stream|.
-			If failure was returned,
+			and the <a>current input code point</a> is not U+007C VERTICAL LINE (|),
 			return failure;
-			otherwise,
-			[=list/append=] the returned value to |descriptor|.
 
-			If |descriptor|’s [=list/size=] is zero, return failure.
+		:: If the <a>current input code point</a> is not U+007C VERTICAL LINE (|),
```

Comment:
These should all be paragraphs under the one dd, not multiple separate dds

Reply:
Sure, done.